### PR TITLE
initial conda recipe for python-fastimport

### PR DESCRIPTION
**Pull request by mikofski**
fastimport is an optional requirement for dulwich that implements a parser/generator for the git-fastimport format

### CODE REVIEW

File: recipes/fastimport/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - python
+
+test:
+  imports:
+    - fastimport
+    - fastimport.tests
+    - fastimport.processors
+
+about:
+  home: htps://github.com/jelmer/python-fastimport
```
Comment (synapticarbors) [inline recipes/fastimport/meta.yaml]: `htps` -> `https`

Comment (synapticarbors) [reply recipes/fastimport/meta.yaml]: There is still a typo in the `home` entry.

**Comment by mikofski** [comment]
@jelmer should fastimport be a requirement of dulwich or an option to be installed separately as desired by the user? Currently it is not required by the conda-forge distribution, although fastimport is available separately. Should there be something in the description for either dulwich or fastimport re: this option?

**Comment by jelmer** [comment]
fastimport is an optional dependency; it's not necessary for most functionality provided by dulwich. you only need it to read and write fastimport streams.

**Comment by mikofski** [comment]
Ok thanks

**Pull request merged**